core/templates/Polyfills.ts

```diff
@@ -139,8 +139,8 @@ if (navigator.mediaDevices.getUserMedia === undefined) {
     var getUserMedia = (
       // This throws "Property 'webkitGetUserMedia' does not exist on
       // type 'Navigator'." This is because this API is deprecated.
-      // (https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getUserMedia)
-      // We need to suppress this because some browsers
+      // (https://developer.mozilla.org/en-US/docs/Web/API/Navigator
+      // /getUserMedia). We need to suppress this error because some browsers
```

Comment:
Ditto, why you changed this?

Reply:
I changed this because it does not follow the pattern required for '@ts-ignore' or '@ts-expect-error' which I am implementing in comment-style check
https://github.com/oppia/oppia/pull/13022/files#diff-ebdbd754a4a10a16d3a71e77edcaa4e50310e3e7e14fbb47a456f5d0d0b4d272R92-R105

Reply:
Thanks for the explanation. :)